__tests__/wallet/partialTxProposal.test.ts

```diff
@@ -354,6 +355,26 @@ test('resetSignatures', async () => {
   expect(proposal.transaction).toEqual(null);
 });
 
+test('setSignatures', async () => {
```

Comment:
💬 In this test it would be nice to have an assert for the `signatures` value, specially after a merge with a different `serializedSignatures`.

Reply:
The signature adding process is quite complex and would require access to the storage, initializing wallets, etc.

The scope of this unit test is just to make sure the correct `addSignatures` method ( mocked here ) is called with the correct parameters, and trust that this method's results are being well tested elsewhere. Also, there is no merge on this method, only overriding of the local signatures.

The [integration tests](https://github.com/HathorNetwork/hathor-wallet-lib/compare/dev...feat/swap-service-integration) that will be added later will make sure all pieces are working together correctly.

Reply:
Nice!